cypress/integration/settings/circulation/fixed-due-date-schedules/schedule-does-not-cover-the-test-date.spec.js

```diff
@@ -0,0 +1,177 @@
+import moment from 'moment';
+import uuid from 'uuid';
+
+import TestTypes from '../../../../support/dictionary/testTypes';
+import SettingsMenu from '../../../../support/fragments/settingsMenu';
+import generateItemBarcode from '../../../../support/utils/generateItemBarcode';
+import getRandomPostfix from '../../../../support/utils/stringTools';
+import {
+  REQUEST_POLICY_NAMES,
+  NOTICE_POLICY_NAMES,
+  OVERDUE_FINE_POLICY_NAMES,
+  CY_ENV,
+  LOAN_TYPE_NAMES,
+  MATERIAL_TYPE_NAMES,
+  LIBRARY_DUE_DATE_MANAGMENT,
+  LOAN_PROFILE,
+  LOST_ITEM_FEES_POLICY_NAMES,
+} from '../../../../support/constants';
+import fixedDueDateSchedules from '../../../../support/fragments/circulation/fixedDueDateSchedules';
+import checkout from '../../../../support/fragments/checkout/checkout';
+import loans from '../../../../support/fragments/loans/loansPage';
+import topMenu from '../../../../support/fragments/topMenu';
+
+let userId;
+let rulesDefaultString;
+const USER_BARCODE = uuid();
+const ITEM_BARCODE = generateItemBarcode();
+const fromDate = moment.utc().subtract(2, 'days');
+const toDate = moment.utc().add(2, 'days');
+const dueDate = moment.utc().add(2, 'days');
+const newToDate = moment.utc().subtract(1, 'days');
+
+describe('ui-circulation-settings: Fixed due date schedules', () => {
+  before(() => {
+    cy.login(Cypress.env(CY_ENV.DIKU_LOGIN), Cypress.env(CY_ENV.DIKU_PASSWORD));
+    cy.getToken(Cypress.env(CY_ENV.DIKU_LOGIN), Cypress.env(CY_ENV.DIKU_PASSWORD))
+      .then(() => {
+        cy.getInstanceTypes({ limit: 1 });
+        cy.getHoldingTypes({ limit: 1 });
+        cy.getLocations({ limit: 1 });
+        cy.getHoldingSources({ limit: 1 });
+        cy.getLoanTypes({ query: `name="${LOAN_TYPE_NAMES.CAN_CIRCULATE}"` });
+        cy.getMaterialTypes({ query: `name="${MATERIAL_TYPE_NAMES.MICROFORM}"` });
+        cy.getUserGroups({ limit: 1 });
+      })
+      .then(() => {
+        cy.createUserApi({
+          active: true,
+          barcode: USER_BARCODE,
+          personal: {
+            preferredContactTypeId: '002',
+            lastName: `Test user ${getRandomPostfix()}`,
+            email: 'test@folio.org',
+          },
+          patronGroup: Cypress.env(CY_ENV.USER_GROUPS)[0].id,
+          departments: [],
+        })
+          .then((user) => {
+            userId = user.id;
+          });
+        cy.createInstance({
+          instance: {
+            instanceTypeId: Cypress.env(CY_ENV.INSTANCE_TYPES)[0].id,
```

Comment:
try to work through test consts instead of Cypress.env

Reply:
Refactored, but not everywhere. Because a lot of values used once but in different scope than where they can be assigned. So don't want to create a lot of `let` variables for these values.